memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
```

Comment:
[PR 523](https://github.com/brianfrankcooper/YCSB/pull/523) moves Java + Maven instructions from MongoDB's README to the top-level README.